README.md

```
@@ -1,35 +1,164 @@
 
 ## Build and run (for developers)
 Refer [Build and Run](docs/build-and-run.md).
-    
+
 ## Deploy
 To deploy print service in production follow the given steps:
 
 1. Onboard your organisation as [Credential Partner](https://docs.mosip.io/1.2.0/partners).
 2. Place your `.p12` file in `../src/main/resources` folder.
-3. Set configuration as in given [here](docs/configuation.md).
+3. Set configuration as in given [here](https://github.com/mosip/mosip-config/blob/release-1.3.x/print-default.properties).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)